cdi/src/main/java/com/oracle/odi/cdi/OdiInstanceImpl.java

```diff
@@ -55,46 +56,67 @@
     private Map<T, CreationalContext<T>> created = new HashMap<>();
 
     OdiInstanceImpl(BeanContext beanContext,
-                    OdiBeanContainer beanContainer,
+                    OdiBeanContainerImpl beanContainer,
```

Comment:
Is there a way not to pass implementations?

Reply:
Will investigate